<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientKredit</class>
 <widget class="QDialog" name="ClientKredit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кредиты клиентов</string>
  </property>
  <widget class="QTableWidget" name="twClientKredit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>581</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectAll">
   <property name="geometry">
    <rect>
     <x>871</x>
     <y>9</y>
     <width>98</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать Все</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>361</width>
     <height>169</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLineEdit" name="leDateClose"/>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QPushButton" name="btnEdit">
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="IDKredit">
      <property name="text">
       <string>Кредит</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLineEdit" name="lePart"/>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="btnDel">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="lbDateClos">
      <property name="text">
       <string>Дата закрытия</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLineEdit" name="leIDKredit"/>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLineEdit" name="leDateOp"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="lbPart">
      <property name="text">
       <string>Погашенная часть</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="lbDateOp">
      <property name="text">
       <string>Дата Открытия</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="leSearch">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>184</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twSec">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>215</y>
     <width>361</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>756</x>
     <y>9</y>
     <width>109</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Подключение</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
